pkg/checks/checks.go

```diff
@@ -71,6 +72,22 @@ func checkNamespace(namespace string) bool {
 
 // WaitForInfraServices verify daemonsets, statefulsets
 func WaitForInfraServices(ctx context.Context, appclient *appclient.AppsV1Client) error {
+out:
+	for {
+		_, err := appclient.Deployments("openshift-web-console").Get("webconsole", metav1.GetOptions{})
```

Comment:
I still believe this helper should check a whitelist of the infra services that we expect to create/update.

Reply:
You're right. Opened #273.